src/pytest/test.py

```diff
@@ -918,9 +918,9 @@ def testGeoErrors(env):
     env.expect('ft.add idx hotel 1.0 fields name hill location -0.1757,51.5156').equal('OK')
     env.expect('ft.search idx hilton geofilter location -0.1757 51.5156 1 km').equal([0L])
 
-    # Insert error
-    env.expect('ft.add', 'idx', 'hotel1', 1, 'fields', 'name', '_hotel1', 'location', '1, 1').error()   \
-            .contains('Could not index geo value')
+    # Insert error - works fine with out of keyspace implementation
```

Comment:
Why is it working fine?

Reply:
because we are using `strtod` and not rely on GEOADD errors.

Reply:
Why would GEOADD will raise error here and we are not?

Reply:
Before we wouyld pass it to redis as `geoadd idx 1 " 1" doc` and will receive error `ERR value is not a valid float`.
Now that we parse it ourselves, `strtod` just ignores it.

Reply:
```
* Note that this function demands that the string strictly represents
 * a double: no spaces or other characters before or after the string
 * representing the number are accepted. */
int string2d(const char *s, size_t slen, double *dp) {
```
https://github.com/antirez/redis/blob/bdd541174736522c92ff786576e744269c3a1c2c/src/util.c#L497

We are using `strtod` everywhere in the parser so I think it is OK.